docs/content/TooltipV2.mdx

```diff
@@ -1,11 +1,18 @@
 ---
 componentId: tooltip_v2
 title: Tooltip v2
-status: Draft
-source: https://github.com/primer/react/tree/main/packages/react/src/drafts/Tooltip.tsx
+status: Beta
+a11yReviewed: true
+source: https://github.com/primer/react/tree/main/packages/react/src/TooltipV2/Tooltip.tsx
 ---
 
-import data from '../../../packages/react/src/drafts/Tooltip/Tooltip.docs.json'
+import data from '../../packages/react/src/TooltipV2/Tooltip.docs.json'
+
+```js
+import {Tooltip} from '@primer/react/next'
+```
```

Comment:
```suggestion
```jsx
//👇 Notice the new bundle, `next`👇
import {Tooltip} from '@primer/react/next'
```
```